src/field_type.cpp

```diff
@@ -63,6 +63,20 @@ int_id<field_type>::int_id( const string_id<field_type> &id ) : _id( id.id() )
 {
 }
 
+const field_intensity_level &field_type::get_intensity_level( int level ) const
+{
+    if( intensity_levels.empty() ) {
```

Comment:
To appease clang builds I am removing emptiness check clause in #33024 - it is already done during json loading anyway and it seems redundant to me here, but please tell me if you have other concerns about that.